common/cache/namespaceCache.go

```diff
@@ -30,16 +30,19 @@ import (
 	"sync/atomic"
 	"time"
 
-	namespacepb "go.temporal.io/temporal-proto/namespace"
+	"github.com/gogo/protobuf/proto"
+	"go.temporal.io/temporal-proto/namespace"
 	"go.temporal.io/temporal-proto/serviceerror"
 
+	"github.com/temporalio/temporal/.gen/proto/persistenceblobs"
```

Comment:
Leave `persistenceblobs` as is for now. It is the only one package which I haven't touched yet.